ui/src/pages/Access/Accounts.tsx

```diff
@@ -1,25 +1,40 @@
-import { access } from '@/api';
+import { access as accessReq } from '@/api';
 import type { AcAccount, AcRole } from '@/api/generated';
 import HandleAccountModal from '@/components/customModal/HandleAccountModal';
 import showDeleteConfirm from '@/components/customModal/showDeleteConfirm';
+import { useAccess } from '@umijs/max';
 import { useRequest } from 'ahooks';
 import type { TableProps } from 'antd';
 import { Button, Space, Table, message } from 'antd';
 import { useState } from 'react';
 import { Type } from './type';
 
-export default function Accounts() {
-  const { data: allAccountsRes } = useRequest(access.listAllAccounts);
-  const allAccounts = allAccountsRes?.data;
+interface AccountsProps {
+  allAccounts: AcAccount[] | undefined;
+  refreshAccounts: () => void;
+}
+
+export default function Accounts({
+  allAccounts,
+  refreshAccounts,
+}: AccountsProps) {
+  const access = useAccess();
```

Comment:
What would `useAccess()` return?

Reply:
The object returned in access.ts. https://umijs.org/docs/max/access#useaccess